### ISSUE TITLE
I have a trouble in the Galleon's Graveyard

### ISSUE THREAD

**Issue by OoYuFanoO** [Wynncraft/Issues#7264]
When I kill the Skeletal Pirate (Lv. 65) to next room  (after the dialogue) It becomes this:
![2017-01-26_22 07 04](https://cloud.githubusercontent.com/assets/25367314/22334043/0f47d70a-e414-11e6-94d5-a9e2dff0dabb.png)
Then not any action!!!!!

**Comment by Pyromanical** [comment]
Are you on 1.9+?  The cutscenes have a habit of getting stuck if you aren't in 1.8.x.  You are going to have to do /kill to get out, as you can't continue.

**Comment by JeffTheNub** [comment]
I don't agree with Pyro. I've experienced this situation every time and I'll just lie backwards and do /class, and you'll slip out from the barrier, it's the only way to escape. I am not abusing glitches as I tried to get in "my" ship and they were not firing at me which equals to no progress, so...this is the only way. Sorry if you were disturbed by my way of doing this part in the Sea Dungeon.

**Comment by JeffTheNub** [comment]
And by the way I've report this bug and I think they may fix it soon. As they are working on the dungeons recently.

**Comment by DebugMC** [comment]
I just tried on 1.8 and got stuck. Also as this is a common problem I think this is just for this dungeon and should get the 'world tag' however, dupelicate of #5853 #7238 #7125 and #6460 all are about the graveyard.

**Comment by OoYuFanoO** [comment]
Thanks you Now I can to next room and to kill the BOSS to get many EXP~  Thanks very much!!!!!

**Comment by Numbermaniac** [comment]
Yep, looks like @JeffTheNub's solution fixes it: go as far back as you can, then /class and select your class again, and you can slip out.
